x/emissions/keeper/actor_penalties_test.go

```diff
@@ -260,7 +332,62 @@ func TestCountWorkerContiguousMissedEpochs(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			missedEpochs := keeper.CountWorkerContiguousMissedEpochs(topic, tc.lastSubmittedNonce)
+			missedEpochs := keeper.CountInactiveWorkerContiguousMissedEpochs(topic, tc.lastSubmittedNonce)
+			if missedEpochs != tc.expectedMissedEpochs {
+				require.Equal(t, tc.expectedMissedEpochs, missedEpochs, "expected %d, got %d", tc.expectedMissedEpochs, missedEpochs)
+			}
+		})
+	}
+}
+
+// nolint: exhaustruct
+func TestCountActiveWorkerContiguousMissedEpochs(t *testing.T) {
+	topic := types.Topic{
+		EpochLastEnded: 105,
+		EpochLength:    10,
```

Comment:
sam ehere about the topic definition validations, which are not run here but may lead to confusion